docs/04_User_Guides/autofix_js_css_resources.md

```diff
@@ -24,14 +18,41 @@ will lint all your css  files and automatically fix the errors
 more info for crafty eslint options here:  https://swissquote.github.io/crafty/Packages/crafty-preset-eslint/index.html
 
 
-# Option2 :  Prettier is used internally with crafty 
+## Option 2 :  integrate crafty linter in idea 
+
+This option permits you to format using your IDE's built-int mechanism 
+more info here : https://swissquote.github.io/crafty/Packages/crafty-preset-eslint/ESLint_IDE_Integration.html
+
+`yarn crafty ide`
+
+## Option 3 :  Prettier is used internally with crafty 
 
 Prettier is used internally with crafty and some times regex paths in eslint have issues: 
 
 so some equivalent commands for javascript linting is: 
 
-- `node_modules/.bin/prettier --write "**/*.ts"`
+Note the `--write ` option is used to save the modifications 
+feel free to ommit it in case you want just to preview.
+
+- For typescript
+	- `yarn prettier --write "**/*.ts"`
+ 	- or `node_modules/.bin/prettier --write "**/*.ts"`
```

Comment:
don't mention node_modules/.bin it adds a lot of noise and not a lot of value

Reply:
replace ./node_modules/bin* with npx

Reply:
replaced*+